README.md

```diff
@@ -17,11 +17,53 @@ A Chef Server cluster utilizing Amazon services for high availability, auto-scal
 
 # Using it
 
-## Requirements:
+## Requirements
 * A working knowledge and comfort level with CloudFormation so that you can read and understand this template for your self
 * Permissions to create all of the types of resources specified in this template (IAM roles, Database subnets, etc)
 * A valid SSL certificate ARN (from the AWS Certificate Manager service)
 
+## Prerequisites
+
+Before you fire it up, there are a few things you should make sure you have prepped and ready.
+
+### AMI Acceptance
+
+Before you start, make sure the AMI that is listed in the `backendless_chef.yaml` for your region has been accepted/approved for use in the AWS Marketplace. We are using native Amazon Linux AMIs, the most up to date list is [always here](https://aws.amazon.com/amazon-linux-ami/).
+
+### For SSL
+1. You must have a DNS Record Name and Zone Name defined prior to setup if you want SSL.
+1. You must have already created and uploaded an SSL cert to AWS, this can only be done via the console at this time. Once you've uploaded the cert, you will need the `ARN` of the cert resource.
+
+### Using with Automate
+
+At this time, this template does not setup Automate as part of it's deployment. However, you can configure it to point to an Automate server that you will setup afterwards, or that is already setup.
+1. You should have a token already generated, follow the [instructions here](https://docs.chef.io/data_collection.html#step-1-configure-a-data-collector-token-in-chef-automate) on how to generate an Automate token.
+1. The Chef Automate URL, for more information on how this URL is formatted, [read this](https://docs.chef.io/data_collection.html#step-2-configure-your-chef-server-to-point-to-chef-automate).
+
+### Package Versions
+
+It's recommended to have the packages downloaded and hosted locally before proceeding, an S3 bucket works well for this purpose. Once you've downloaded the correct EL7 packages and have them hosted, adjust the following variables accordingly to point to the proper URLs.
```

Comment:
You know what would be awesome here?  a few lines of shell magic to show how to do this using `mixlib-install` and `aws s3 sync`

Reply:
Good idea, haven't used mixlib-install yet, good future PR :)